worlds/inscryption/Options.py

```diff
@@ -109,9 +103,8 @@ class PaintingChecksBalancing(Choice):
     option_force_filler = 2
 
 @dataclass
-class InscryptionOptions(PerGameCommonOptions):
-    deathlink: DeathLink
-    act1_deathlink_behaviour: Act1DeathLinkBehaviour
+class InscryptionOptions(DeathLinkMixin, PerGameCommonOptions):
+    act1_death_link_behaviour: Act1DeathLinkBehaviour
```

Comment:
```suggestion
from Options import DeathLink
...

class InscryptionOptions(DeathLinkMixin, PerGameCommonOptions):
    death_link: DeathLink
    act1_death_link_behaviour: Act1DeathLinkBehaviour
```

Reply:
Doesn't DeathLinkMixin already add death_link to this list automatically? I used The Messenger as reference: https://github.com/ArchipelagoMW/Archipelago/blob/1e3a4b6db5acff8e82bdecdc79773f2923de919a/worlds/messenger/options.py#L221

Reply:
I have no idea how DeathLinkMixin works, I just know you removed death link from your options so I'd assume that would break it?

Reply:
Judging from other implementations and from how death_link appears as an attribute in the class without adding it myself, it seems to work as it is currently.

Reply:
Ah, my bad then